.github/workflows/human_pull_request.yml

```diff
@@ -1,22 +0,0 @@
-name: pull request
-
-on:
-  pull_request:
```

Comment:
we can set this to `on: push: branches-ignore: [dependabot/*]` so it doesn't show up at all for dependabot prs

Reply:
nvm this doesn't work in this case because env vars aren't being set correctly (`GITHUB_BASE_REF`) for the action to work

Reply:
Ya, I think we had tried that a while ago. I looked for some way to filter on the PR branch, but the docs say there is none. Has to be an `if`.

Reply:
Unfortunately GitHub Actions is lacking on this front. GitLab allows you to reference both the HEAD ref and the BASE ref for merge requests